labgrid/resource/udev.py

```diff
@@ -301,11 +301,12 @@ def update(self):
         if not self.device:
             self.control_path = None
             self.disk_path = None
```

Comment:
I'd prefer a `return` here.

Reply:
Can you elaborate on why?

Reply:
I would assume to reduce unnecessary nesting depth for conditionals.
Sounds legitimate.

Reply:
Correct, drop the else and return on `not self.device`.